packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -179,7 +180,7 @@ const buildEventsData = ({
       }
 
       // Pass app data that works with routing forms
-      const eventTypeAppMetadata = getEventTypeAppMetadata(eventType.metadata);
```

Comment:
As `getEventTypeAppMetadata` reparses the metadata assuming it as JSON value, this issue started happening.